perun-core/src/main/java/cz/metacentrum/perun/core/impl/ExtSourceCSV.java

```diff
@@ -282,4 +287,13 @@ private boolean compareRowToQuery(String[] row, String query) throws InternalErr
         }
         return lineAsMap;
     }
+
+	protected Map<String,String> getAttributes(ExtSource extSource) {
```

Comment:
This is just the copy of the same method in every ExtSource, better use the one in BlImpl of ExtSource manager. If needed, let Spring to fill object perun to these extSources and call it by perun.getExtSourcesManagerBl....